hc/settings.py

```diff
@@ -129,7 +129,8 @@
 
 USE_TZ = True
 
-SITE_ROOT = "https://health-checks-wits.herokuapp.com"
+# SITE_ROOT = "https://health-checks-wits.herokuapp.com"
+SITE_ROOT="http://localhost:8000"
```

Comment:
Move this variable to the environment.

Reply:
@john555  I seem not to see the uncommented `SITE_ROOT` variable locally, let me go ahead and remove the commented one, it's the only one I can see. Then I'll push again.